### PR TITLE
Add Bootstrap Spinner Component

### PR DESCRIPTION
**Pull request by onechiporenko**
Add support for Spinner Component ([BS Docs](https://getbootstrap.com/docs/5.2/components/spinners/#about)).

### CODE REVIEW

File: addon/components/bs-spinner.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ ### Advanced Usage
+
+ ```hbs
+ <BsSpinner @spinnerType="grow" @size="sm" @type="warning"/>
+ ```
+
+ ### Usage with `BsButton`
+
+  ```hbs
+  <BsButton @onClick=(action "download") as |button|>
```
[the remark this answers is not in the document]
Comment (onechiporenko) [reply addon/components/bs-spinner.js:23]: Nope, just copied from `bs-button`.

File: addon/components/bs-spinner.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import Component from '@glimmer/component';
+import arg from '../utils/decorators/arg';
+
+/**
+ Implements a [Spinner element](https://getbootstrap.com/docs/5.2/components/spinners/).
+
+ ### Basic Usage
+
+ ```hbs
+ <BsSpinner/>
```
Comment (simonihmig) [inline addon/components/bs-spinner.js:10]: With the recent changes I would like to have the accessible version shown here:

```suggestion
 <BsSpinner>Loading...</BsSpinner>
 
 The (optional) component's block will render as visually invisible but accessible text for assistive technology like screen readers.
```

File: addon/components/bs-spinner.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  @arg
+  spinnerType = 'border';
+
+  /**
+   * @property accessibilityText
+   * @type string
+   * @public
+   */
+  @arg
+  accessibilityText = 'Loading...';
```
[the remark this answers is not in the document]
Comment (simonihmig) [reply addon/components/bs-spinner.js:74]: I agree.

As this has already been addressed, @onechiporenko this property and its doc blocks can be removed I think!

File: docs/app/templates/demo/spinner.hbs [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+<h2>
+  Options
+</h2>
+<div class="bs-example">
+  <!-- BEGIN-SNIPPET spinner-options -->
+  <p>
+    <BsSpinner @size="sm"/>
+  </p>
+  <p>
+    <BsSpinner @type="danger"/>
```
Comment (simonihmig) [inline docs/app/templates/demo/spinner.hbs:21]: Can you please add the `Loading...` test here also in these examples!

**Review by simonihmig — APPROVED** [review]
LGTM, thanks again!

**Pull request merged**